### PR TITLE
feat: coin support tron with create and receive operation

### PR DESCRIPTION
**Pull request by jhasuraj01**
Tickets:
- PRF-7203
- PRF-7205

### CODE REVIEW

File: packages/coin-support-tron/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@cypherock/coin-support-tron",
+  "version": "0.0.5",
```
Comment (irshadjsr21) [inline packages/coin-support-tron/package.json]:
```suggestion
  "version": "0.0.0",
```

File: packages/coin-support-tron/src/types.d.ts
Comment (irshadjsr21) [inline packages/coin-support-tron/src/types.d.ts:1]: The `cysyncEnv` is not used anywhere in any `coin-support-*` packages. I think we can safely remove it

File: packages/coins/src/tron/index.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import coinList from './coins';
+
+import { ICoinInfo, coinFamiliesMap } from '../types';
+
+type TronFamily = typeof coinFamiliesMap.tron;
+
+export interface ITronCoinInfo extends ICoinInfo {
+  family: TronFamily;
+  apiCoinType: string;
```
Comment (irshadjsr21) [inline packages/coins/src/tron/index.ts]: I think `apiCoinType` won't be needed in case of tron

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]

File: packages/coin-support-tron/src/types.d.ts
Comment (jhasuraj01) [reply packages/coin-support-tron/src/types.d.ts:1]: done

File: packages/coins/src/tron/index.ts (hunk 2 above)
Comment (jhasuraj01) [reply packages/coins/src/tron/index.ts]: done

**Review by irshadjsr21 — APPROVED** [review]

**Pull request merged**